repose-aggregator/components/filters/openstack-identity-v3/src/main/scala/org/openrepose/filters/openstackidentityv3/utilities/OpenStackIdentityV3API.scala

```diff
@@ -1,19 +1,22 @@
 package org.openrepose.filters.openstackidentityv3.utilities
```

Comment:
As I discovered the myriad of String concatenation, I updated with interpolation.